### ISSUE TITLE
Snap produces `relocation error` on Ubuntu 14.04.5

### ISSUE THREAD

**Issue by codyhazelwood** [minio/minio#5472]
## Expected Behavior

Minio should start, or produce an error message as to why it couldn't.

## Current Behavior

When running `snap run minio server ...`, produces:
`minio: relocation error: /snap/minio/816/lib/x86_64-linux-gnu/libpthread.so.0: symbol __mmap, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference`

## Steps to Reproduce (for bugs)

1. `sudo apt-get update && sudo apt-get upgrade && sudo apt-get install snapd`
2. `sudo reboot`
3. `sudo snap install minio --edge`
4. `sudo mkdir /test`
5. `sudo snap connect minio:mount-observe`
6. `sudo snap run minio server /test`

## Your Environment

* Version used (`minio version`):  snap master rev 816
* Environment name and version (e.g. nginx 1.9.1):
* Server type and version: Ubuntu Server 14.04.5 running in a Parallels VM
* Operating System and version (`uname -a`): `Linux local-dev 4.4.0-111-generic #134~14.04.1-Ubuntu SMP Mon Jan 15 15:39:56 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`
* Link to your project:

**Comment by harshavardhana** [comment]
Ubuntu snaps are for latest releases only i.e Ubuntu 17.10 above. For 14.04 simply download our signed static binaries

**Comment by harshavardhana** [comment]
Closing this as a non-issue, binaries are preferred on older Ubuntu installations.

**Comment by Darkless012** [comment]
Can you please update the Docs accordingly?

It would be nice to update Docs like: 
"Snap install is recommended for latest versions of Linux distributions only. It might fail while running on older distributions."
And maybe provide a table?

Because snap is by default installed on Ubuntu 16.04 LTS (https://docs.snapcraft.io/core/install-ubuntu) so it lead me that finally I can, as the slogan goes: "pack once, run everywhere". 

I was thrived to install this via snap. And I went from "Wow I can install this via snap, finally first app that convinces me to use snap. SNAP fetch it" to "Oh Crap this is the last time I'm using something new."

**Comment by dragon2611** [comment]
Was hoping to install the Minio Snap on the same machine as nextcloud so I could use an S3 client to upload instead of webdav/nextcloud since NC tends to choke on larger files.

I plugged mount-observe and also removeable-media 

Same error as original poster, but I'm on 17.10
```

root@nc:~# snap run minio server /media/disk2/
minio: relocation error: /snap/minio/1186/lib/x86_64-linux-gnu/libpthread.so.0: symbol __mmap, version GLIBC_PRIVATE not defined in file libc.so.6 with link time reference
```

```
root@nc:~# snap info minio
name:      minio
summary:   Open source object storage server compatible with Amazon S3 APIs
publisher: minio-dev
license:   unknown
description: |
  Minio is an object storage server released under Apache License v2.0. It is
  compatible with Amazon S3 cloud storage service. It is best suited for
  storing unstructured data such as photos, videos, log files, backups and
  container / VM images. Size of an object can range from a few KBs to a
  maximum of 5TB.
snap-id: ili7qD2zqHW5piPwr7ZOVezId9vvQpG7
commands:
  - minio
tracking:    edge
installed:   master (1186) 11MB -
refreshed:   2018-03-19 19:50:35 +0000 UTC
channels:                  
  stable:    –                  
  candidate: –                  
  beta:      –                  
  edge:      master (1186) 11MB -
root@nc:~# ^C
root@nc:~# 

```
```
root@nc:~# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=17.10
DISTRIB_CODENAME=artful
DISTRIB_DESCRIPTION="Ubuntu 17.10"

```
